docs-kits/kits/knowledge-agents/development-view/api.md

```diff
@@ -3,7 +3,7 @@ sidebar_position: 1
 title: API
 ---
 <!--
- * Copyright (c) 2021,2023 T-Systems International GmbH
+ * Copyright (c) 2021,2024 T-Systems International GmbH
  * Copyright (c) 2021,2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG) 
  * Copyright (c) 2021,2023 Mercedes-Benz AG
  * Copyright (c) 2021,2023 ZF Friedrichshafen AG
```

Comment:
```suggestion
 * Copyright (c) 2021 ZF Friedrichshafen AG
```

The TRG 7.02 for the Copyright and License Header has been updated slightly. Only the year of the first contribution is referenced as the contribution details can always be derived from the Github commit log (for details see: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02).

Please update the affected licence-header accordingly.

Reply:
I adopted the headers (and visible subscripts).
I also backported all changes to the already "branched" 2412 versioned folder.